proxstar/static/js/script.js

```diff
@@ -223,7 +223,7 @@ $("#create-vm").click(function(){
     const max_cpu = $(this).data('max_cpu');
     const max_mem = $(this).data('max_mem');
     const max_disk = $(this).data('max_disk');
-    var ssh_regex = new RegExp("ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?")
+    var ssh_regex = new RegExp("ssh-[a-z0-9]+ AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?");
```

Comment:
this won't match security keys created with a hardware auth key (yubikey, solokey, etc)

Reply:
who asked?

Reply:
![image](https://github.com/ComputerScienceHouse/proxstar/assets/7281449/f1d570a8-53d5-40a6-afb4-40f11b7a738a)